kuma/javascript/src/payments/pages/management.jsx

```diff
@@ -0,0 +1,188 @@
+// @flow
+import * as React from 'react';
+import { useContext, useEffect, useState } from 'react';
+
+import { gettext, interpolate, Interpolated } from '../../l10n.js';
+import UserProvider, { type UserData } from '../../user-provider.jsx';
+import Subheader from '../components/subheaders/index.jsx';
+import CancelSubscriptionForm from '../components/cancel-subscription-form.jsx';
+import ErrorMessage from '../components/error-message.jsx';
+import { getSubscriptions } from '../api.js';
+import { formatDate } from '../formatDate.js';
+
+export type SubscriptionData = {
+    id: string,
+    amount: number,
+    brand: string,
+    expires_at: string,
+    last4: string,
+    zip: string,
+    next_payment_at: string,
+};
+
+type Props = {
+    locale: string,
+};
+export const title = gettext('Manage monthly subscriptions');
+export const successMsg = gettext(
+    'Your monthly subscription has been successfully canceled.'
+);
+
+const ManagementPage = ({ locale }: Props) => {
```

Comment:
I don't yet know what's causing this but there's definitely something strange going on with the loading order on this page. 

This is what I get when I refresh the page. 
![Screen-Recording-2020-04-24-at-2 38 22-PM](https://user-images.githubusercontent.com/26739/80246364-4873df80-863a-11ea-80b8-61cc58bd8472.gif)